### ISSUE TITLE
Use Allison's biology m-files as functions

### ISSUE THREAD

**Issue by amandafaig** [mspringborn/salmonids#3]
1. check files and edit to ensure compatibility with FDP scripts
2. ensure generalizable for one and two reach scenarios
2. integrate into code

**Comment by amandafaig** [comment]
This is partway done.  It's stalled because Allison & Marissa use fishing pressure as a rate, rather than as a percent of total population. (So annual fishing is 1-exp(-F) rather than (1-escapement).  Are we OK with switching to a rate, or should I ask them to switch?

**Comment by mspringborn** [comment]
I don't quite understand the situation. Are you saying that they specify harvest in terms of a number of fish taken and we have a percentage of the stock taken (or equivalently left)?

If so, why is that a problem? We know the stock level perfectly at harvest so there is a one to one mapping between harvest as a raw number and harvest as a percentage.

**Comment by amandafaig** [comment]
Nope,  I’m saying they specify it as an intensity measure.  

So if we say we want escapement to be 20%, they find the level of intensity with which exp(-F) = 20%.  F is an instantaneous fishing mortality, so  1-exp(-F) is the discrete approximation.

You’re right, there’s no huge problem, since there’s a formula that will translate: it’s just that we need to figure if we want to use out one or the other.  I suppose I could write a line of code that finds F given escapement so we can pass F through and use the code as written, but it seems like the better long term solution is to just be on the same page?

-------------------------------------------------------------------
Amanda Faig
Ph.D. Candidate
Agricultural and Resource Economics, UC Davis
530 848 0938
afaig@ucdavis.edu

**Comment by mspringborn** [comment]
Yes, better to have it in terms of harvested number or percentage.

Mike Springborn
UC Davis
http://springborn.faculty.ucdavis.edu/



-----Original Message-----
